Dockerfile

```diff
@@ -36,7 +36,6 @@ ARG TARGETARCH
 RUN --mount=type=cache,target=/go/pkg/mod/ \
     --mount=type=cache,target="/root/.cache/go-build" \
     CGO_ENABLED=0 GOOS=linux GOARCH=${TARGETARCH} GO111MODULE=on make build-operator
-RUN setcap 'cap_net_bind_service=+ep' /workspace/bin/spark-operator
```

Comment:
you can also remove lines 23-25, since we only install `libcap2-bin` to be able to run `setcap`

Reply:
Thanks, good catch I missed that